app/views/docs/rest.phtml

```diff
@@ -1,12 +1,377 @@
 <p>Appwrite supports multiple protocols for accessing the server, including <a href="/docs/rest">REST</a>, <a href="/docs/graphql">GraphQL</a>, and <a href="/docs/realtime">Realtime</a>.</p>
 
-<p>The REST API allows you to manipulate your Appwrite server through many endpoints, each representing a specific method of modification to a specific resource type.</p>
+<p>The REST API allows you to access your Appwrite server through HTTP requests, without the need for an SDK. Each endpoint in the API represents a specific operation on a specific resource.</p>
 
-<p>Requests and responses through the REST API follow a rigid structure - that is, they always expect the same structure for requests to a specific endpoint, and responses from that endpoint will also always have the same structure. </p>
+<h2><a href="/docs/rest#headers" id="headers">Headers</a></h2>
+<p>Appwrite's REST APIs expect certain headers to be included with each request:</p>
 
-<p>The specific structures used for Appwrite resources and their related endpoints can be found in the <b>References</b> section for each service.</p>
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 300px">Header</td>
+        <td style="width: 80px"></td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>X-Appwrite-Project: [PROJECT-ID]</code></td>
+        <td><span class="tag red">required</span></td>
+        <td>Provides the ID of your Appwrite Project to the REST API.</td>
+    </tr>
+    <tr>
+        <td><code>Content-Type: application/json</code></td>
+        <td><span class="tag red">required</span></td>
+        <td>Declares content type of the HTTP request.</td>
+    </tr>
+    <tr>
+        <td><code>X-Appwrite-Key: [API-KEY]</code></td>
```

Comment:
Oops I forgot about `X-Appwrite-JWT` header. Maybe that should be added here and a section in Client Authentication about it?

Reply:
Added. We'll be adding a section in auth guide about JTW in the future, but for now, I added some basic information about JWT here. We'll be able to remove some of this and just link to that guide for details later.